daemon/api.go

```diff
@@ -2265,10 +2265,12 @@ func changeAliases(c *Command, r *http.Request, user *auth.UserState) Response {
 	if err := decoder.Decode(&a); err != nil {
 		return BadRequest("cannot decode request body into an alias action: %v", err)
 	}
-	if len(a.Aliases) == 0 {
-		return BadRequest("at least one alias name is required")
+	if len(a.Aliases) != 0 {
+		return BadRequest("cannot interpret request, snaps can no longer be expected to declare their aliases")
```

Comment:
yes, one of the next PRs needs to bring some form of this API back but probably we still need to use different fields or type, and be able to get a clean error for the previous usage